### ISSUE TITLE
Remove docker dependency

### ISSUE THREAD

**Issue by rverpillot** [aws/copilot-cli#3639]
We don't use copilot to build docker images (images are built in another CI process), but copilot fails on `copilot svc init` because docker is missing: `Cannot connect to the Docker daemon at unix:///var/run/docker.sock`

**Comment by paragbhingre** [comment]
Hello @rverpillot,

Thank you for reporting the issue with us. 

Could you please let us know how you are trying to set the image location when you do `copilot svc init`? 
You can use `copilot svc init --image` to set the image which will solve the problem. 

Let us know if you face any issues after trying above solution.

**Comment by rverpillot** [comment]
We use `image.location` to reference  the docker images in the manifest file, so we don't need `docker` at all because images are already built and deployed into ECR or other repositories. 

But `copilot` currently needs a docker installation and fails without it.

So I understand that `copilot` uses docker when you use `image.build`, but docker shouldn't be a requirement when you use `image.location`.

Maybe this error occurs only when the manifest file already exists when doing `copilot svc init` (we don't use this step to create the manifest) ?

**Comment by efekarakus** [comment]
Hi @rverpillot !

@paragbhingre  Looks like the issue can be reproduced if you just keep pressing Enter (leave the image location as an empty string)
```
$copilot svc init                                                                                                                                                                                               
Note: It's best to run this command in the root of your workspace.
Service type: Load Balanced Web Service
Service name: issue-3639
Docker daemon is not responsive; Copilot won't build from a Dockerfile.
Image: 
Port: 80
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
✘ get docker engine platform: run docker version: exit status 1
```

This is because we hit this if-statement:
https://github.com/aws/copilot-cli/blob/0bea71053c1f260ad4f832d7cb550353941f5d51/internal/pkg/cli/svc_init.go#L290-L292

Another way of triggering the issue is if the manifest file already exists although the service wasn't created yet with init:
```
$ copilot svc init 
Note: It's best to run this command in the root of your workspace.
Service type: Backend Service
Service name: backend
Manifest file for service backend already exists. Skipping configuration.
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
✘ get docker engine platform: run docker version: exit status 1
```

**Comment by efekarakus** [comment]
This fix is now out in v1.20 🚀 
Release notes: https://github.com/aws/copilot-cli/releases/tag/v1.20.0 !
Blog post: https://aws.github.io/copilot-cli/blogs/release-v120/
